src/action/linux/provision_determinate_nix_shim/nix-daemon.determinate-nix.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Nix Daemon, with Determinate Nix superpowers.
+Documentation=man:nix-daemon https://determinate.systems
+RequiresMountsFor=/nix/store
+RequiresMountsFor=/nix/var
+RequiresMountsFor=/nix/var/nix/db
+ConditionPathIsReadWrite=/nix/var/nix/daemon-socket
+
+[Service]
+ExecStart=@/nix/determinate/determinate-nix-ee
+KillMode=process
+LimitNOFILE=1048576
+TasksMax=1048576
```

Comment:
I wonder if one or both of these should be `unlimited` instead?

https://github.com/NixOS/nix/pull/10903#issuecomment-2178998019

Reply:
These specific values shouldn't be unlimited, probably. Making the stack unlimited is a different question, probably to be raised with Eelco.